client/src/components/QuizInstructions/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react"
+import Button from "../sharedComponent/Button"
+import Card from "../Card"
+import ProgressBar from "../sharedComponent/ProgressBar"
+import image from "../../assets/quizInst.png"
+
+class QuizInstructions extends Component {
+  state = {
+    counter: 0,
+    instructions: [
+      "Attention Training wants to help you live better. The first step is understanding whether or not you may have the superpowers of ADHD!",
+      "In order to do this, we’ll ask a few questions to get to know you better. Don’t worry! It only takes about 3 minutes to complete.",
+      "After you finish the quiz, we will provide you with the resources you need to harness your superpowers!",
+      "Ready to get you know your superpowers?"
+    ]
+  }
+  a = <p style={{ "font-size": "48px", "font-weight": "bold" }}> Ready to get you know your superpowers?</p>
+  render() {
+    return (
+      <>
+        <div style={{ height: "42px", display: "flex" }}>
+          <Button
+            margin="18px 0px 4px 8px"
+            hoverColor="#70B1FA"
+            background="#70B1FA"
+            color="#000"
+            border_radius="6px"
+            width="30px"
+            padding="3px 10px 0px 10px"
+            to="/dashboard"
+          >
+            skip
+          </Button>
+          <ProgressBar></ProgressBar>
+        </div>
+        <a href={this.state.counter == 4 ? "/quiz" : null}>
+          <Card
+            onClick={() => {
+              this.setState({ counter: this.state.counter + 1 })
```

Comment:
we can add the logic to change view here.
```suggestion
              this.setState({ counter: this.state.counter + 1 }, () => {
                if (this.state.counter === 4) this.props.history.push('/quiz');
              });
```

We can also use ```componentDidUpdate``` as well. If you feel like using it (i recommend that you do) read up on it in the docs and try it out.